packages/document/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "rspress dev",
-    "build": "rspress build",
+    "dev": "cross-env RUST_MIN_STACK=4194304 rspress dev",
+    "build": "cross-env RUST_MIN_STACK=4194304 rspress build",
```

Comment:
Could this be a common error in rspress? Do we need to set a built-in `RUST_MIN_STACK` value in rspress?

Reply:
For this package, yes as long as upgraded to 0.7.1